recipes/madagascar/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "madagascar" %}
+{% set version = "2.1b.20190205" %}
+{% set commit_hash = "81c978415b3d55769ce92fb37af5c44f65e0eadb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ahay/src/archive/{{ commit_hash }}.zip
+  sha256: e7f0795c166cc7a0a07e98f0996e7dfc88479f562378055a35643e03a7c16be0
+  patches:
+    - SConstruct.patch
+    - configure.patch
+
+build:
+  number: 1
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - scons
+    - swig
+  host:
+    - python
+    - numpy
+    - openblas
+  run:
+    - python
+    - scons
+    - numpy
+    - openblas
+
+test:
+  imports:
+    - rsf.api
+    - c_m8r
+  commands:
+    - test -f $PREFIX/bin/sfwiggle                 # [unix]
+    - if not exist %PREFIX%\\bin\\sfwiggle exit 1  # [win]
+
+about:
+  home: http://www.ahay.org/wiki/Main_Page
+  license: GPL
+  license_file: COPYING.txt
+  summary: "A package for reproducible geophysical data processing"
+
+extra:
+  recipe-maintainers:
+    - dmitry-kabanov (kabanov.dmitry@gmail.com)
```

This should just be your GitHub username.
```suggestion
    - dmitry-kabanov
```